### ISSUE TITLE
Add `schema_version` to `box.status` broadcast

### ISSUE THREAD

**Issue by Serpentian** [tarantool/tarantool#10546]
https://github.com/tarantool/tarantool/issues/10520 will allow users to apply the majority of the DDL during upgrade without any problems: user will have to wait for rw status (via watcher or some config role function) and create their spaces and indexes. The problem there, that some of the DDL will still be forbidden, until box.schema.upgrade is called (e.g. creating persistent triggers) and we should give users a convenient way to use such features.

Currently they will have to check schema version in `_schema`, then `on_replace` trigger for `_schema` will have to be set, and users will have to unregister that trigger after its execution. Moreover, this trigger will check if instance is rw. Not very convenient.

---

Let's  add new field in `box.status`: `schema_version`. In order to apply DDL after Tarantool upgrade, user will create watcher, which checks rw status and schema_version.

This will simplify the realization of https://github.com/tarantool/tarantool/issues/10538

--- 

The alternative of introducing completely new watcher, which includes only `schema_version`, was rejected as Tarantool doesn't allow to create watcher on several events. It will be difficult to synchronize two events: `box.status`, which includes `read_only`, and `box.schema_version`.

**Comment by R-omk** [comment]
https://github.com/tarantool/tarantool/pull/6960/files#diff-f2217bf9a77b1ef1a684973d056602d3321d763b06064c8301f6216d95fbc994R251

**Comment by R-omk** [comment]
> difficult to synchronize two events: 

Just get data via
box.watch_once(key) 
Inside common  callback shared on multiple watcher

**Comment by Serpentian** [comment]
> https://github.com/tarantool/tarantool/pull/6960/files#diff-f2217bf9a77b1ef1a684973d056602d3321d763b06064c8301f6216d95fbc994R251

This is not the schema version, we need. It's a number that indicates whether the database schema is changed. We need version from the `box.space._schema`. 

> Just get data via
box.watch_once(key)
Inside common callback shared on multiple watcher

Yeah, this `box.watch_once` will be on `box.status`. However, this `box.status` doesn't have any info about version of `_schema`. Since in the future some DDL will be available only after schema upgrade, it will be handy to add to `box.status` new field, so that user can write smth like:

```lua
box.watch_once("box.status", function(status)
    if !status.is_ro then
        <create spaces, indexes, some basic DDL>
    end
end)

box.watch_once("box.status", function(status)
    if !status.is_ro and status.version > '3.0.0' then
        <some sophisticated watcher, e.g. creating persistent trigger, which will be available since 3.0.0 schema>
    end
end)
```
